### PR TITLE
Added the environment variables Improve Dockerfile with env variables #800

### PR DESCRIPTION
**Pull request by manjarisri**
# Description

I have added the environment variables in the Dockerfile
Related to issue #800 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [.] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by pieroit** [comment]
Thanks @manjarisri !
Reviewing and merging soon ;)

File: core/Dockerfile [hunk 1]
```diff
@@ -26,5 +25,9 @@ COPY ./cat /app/cat
 COPY ./install_plugin_dependencies.py /app/install_plugin_dependencies.py
 RUN python3 install_plugin_dependencies.py
 
+### ENVIRONMENT VARIABLES ###
+ENV PYTHONUNBUFFERED=1
```
Comment (pieroit) [inline core/Dockerfile:29]: Please move those two lines up, just afet the `FROM`

Comment (pieroit) [reply core/Dockerfile:29]: Additional changes:
- please redo the PR branching off `develop` and towards `develop` (`main` is only for release)
- also update `docker-compose.yml` and take away the unnecessary env vars `PYTHONUNBUFFERED` and `WATCHFILES_FORCE_POLLING`

**Review by pieroit — CHANGES_REQUESTED** [review]

**Comment by manjarisri** [comment]
sure @pieroit

**Comment by manjarisri** [comment]
hey @pieroit, i have raised a new PR, please check

thanks

**Pull request closed without merging**